<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dhinu" failures="4" tests="4" name="Parallel Test" time="7.016" errors="0" timestamp="2024-07-14T22:47:42 IST">
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="4.024">
    <failure type="java.lang.AssertionError" message="expected [111.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [111.0] but found [0.0]
at org.calculator.Calculator_Project.Operations.performOperation(Operations.java:66)
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:33)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="4.024">
    <failure type="java.lang.AssertionError" message="expected [236.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [236.0] but found [0.0]
at org.calculator.Calculator_Project.Operations.performOperation(Operations.java:66)
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:33)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="1.189">
    <failure type="java.lang.AssertionError" message="expected [104.5] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [104.5] but found [0.0]
at org.calculator.Calculator_Project.Operations.performOperation(Operations.java:66)
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:33)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
  <testcase classname="org.calculator.Calculator_Project.CalculatorTests" name="calculate" time="1.169">
    <failure type="java.lang.AssertionError" message="expected [17.0] but found [0.0]">
      <![CDATA[java.lang.AssertionError: expected [17.0] but found [0.0]
at org.calculator.Calculator_Project.Operations.performOperation(Operations.java:66)
at org.calculator.Calculator_Project.CalculatorTests.calculate(CalculatorTests.java:33)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- calculate -->
</testsuite> <!-- Parallel Test -->
